server/src/authorizers/permissions.js

```diff
@@ -34,8 +34,8 @@ const rules = {
     measurements: allow,
     measurable: allow,
     measurables: allow,
-    bot: allow,
-    bots: allow,
+    bot: isAuthenticated,
+    bots: isAuthenticated,
```

Comment:
Why this restriction?

Reply:
Because User can get only its own bots. This is why User should be authenticated.

Reply:
Fixed.